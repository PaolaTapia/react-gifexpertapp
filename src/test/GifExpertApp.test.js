import React from 'react';
import {shallow} from 'enzyme'; 

import GifExpertApp from '../GifExpertApp'; 

describe('Pruebas en GifExpertApp', () => {

     test('debe mostrar el componente correctamente', () => {
          const wrapper= shallow(<GifExpertApp/>); 
          expect(wrapper).toMatchSnapshot();           
     }); 

     test('debe de mostrar una lista de categorÃ­as', () => {
          const categories=['autos', 'cartas']; 
          const wrapper=shallow(<GifExpertApp defaultCategories={categories}/>); 
          expect(wrapper).toMatchSnapshot(); 
          expect(wrapper.find('GifGrid').length).toBe(categories.length);          
     })
     
     
     
})
