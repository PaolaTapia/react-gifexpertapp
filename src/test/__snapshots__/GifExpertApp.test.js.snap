// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en GifExpertApp debe de mostrar una lista de categor√≠as 1`] = `
<Fragment>
  <h2>
      GifExpertApp 
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <GifGrid
    category="autos"
    key="autos"
  />
  <GifGrid
    category="cartas"
    key="cartas"
  />
</Fragment>
`;

exports[`Pruebas en GifExpertApp debe mostrar el componente correctamente 1`] = `
<Fragment>
  <h2>
      GifExpertApp 
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
</Fragment>
`;
